shared/arbiter.py

```
@@ -0,0 +1,202 @@
+        if phase1:
+            return False
+
+        # Phase 2：占坑
+        try:
+            await bot.set_msg_emoji_like(
+                message_id=mid,
+                emoji_id=self._EMOJI_ID,
+                emoji_type=self._EMOJI_TYPE,
+                set=True,
+            )
+        except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)